app/components/timelineButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "@remix-run/react";
+
+export interface TimelineButtonProps {
+  link: string;
+  linkText: string;
+}
+
+export const TimelineButton = ({ link, linkText }: TimelineButtonProps) => {
+  return (
+    <>
+      <Link to={link}>
+        <div>
+          <p>{linkText}</p>
+        </div>
+      </Link>
+    </>
```

Comment:
```suggestion
    <Link to={link}>
      <div>
        <p>{linkText}</p>
      </div>
    </Link>
```
この `Fragment` は不要に思います.